devfiles/java-web-vertx/devfile.yaml

```diff
@@ -0,0 +1,65 @@
+---
+apiVersion: 1.0.0
+metadata:
+  name: java-web-vertx
+projects:
+  -
+    name: java-web-vertx
+    source:
+      type: git
+      location: "https://github.com/che-samples/web-java-vertx"
+components:
+  -
+    type: chePlugin
+    id: redhat/java/latest
+  -
+    type: dockerimage
+    alias: maven
+    image: registry.centos.org/che-stacks/centos-jdk8
+    command: ['sleep']
+    args: ['infinity']
+    env:
+      - name: MAVEN_CONFIG
+        value: /home/user/.m2
+      - name: MAVEN_OPTS
+        value: "-XX:MaxRAMPercentage=50.0 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10
+          -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90
+          -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom
+          -Duser.home=/home/user"
+      - name: JAVA_OPTS
+        value: "-XX:MaxRAMPercentage=50.0 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10
+          -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90
+          -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom"
+      - name: JAVA_TOOL_OPTIONS
+        value: "-XX:MaxRAMPercentage=50.0 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10
+          -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90
+          -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom"
+      - name: PS1
+        value: $(echo ${0})\\$
+      - name: HOME
+        value: /home/user
+    memoryLimit: 512Mi
+    endpoints:
+      - name: '8080/tcp'
+        port: 8080
+    mountSources: true
+    volumes:
+      - name: m2
+        containerPath: /home/user/.m2
+commands:
+  -
+    name: maven build
+    actions:
+      -
+        type: exec
+        component: maven
+        command: "mvn -Duser.home=${HOME} clean install"
+        workdir: "${CHE_PROJECTS_ROOT}/java-web-vertx"
```

Comment:
`java-web-vertx` is confused with `web-java-vertx`

Reply:
can you explain what do you mean?

Reply:
Project is cloned to `web-java-vertx` directory but working dir is `java-web-vertx`

Reply:
which is misbehaving clone issue. It should be cloned to project's name directory `java-web-vertx` and then this command will work.